src/Service/ProgrammesAggregationService.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Service;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ClipRepository;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+use BBC\ProgrammesPagesService\Domain\Entity\CoreEntity;
+use BBC\ProgrammesPagesService\Domain\Entity\Programme;
+use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\CoreEntityMapper;
+use BBC\ProgrammesPagesService\Cache\CacheInterface;
+use InvalidArgumentException;
+
+class ProgrammesAggregationService extends AbstractService
+{
+    const AGGREGATION_VALID_TYPES = [
+        'Series',
+        'Episode',
+        'Clip',
+        'Franchise',
```

Comment:
You don't have `findDescendantFranchise()`, `findDescendantGallery()`, `findDescendantCollection()` or `findDescendantSeason()` methods. Is that correct?